src/components/notice/Notice.stories.tsx

```diff
@@ -99,4 +99,17 @@ stories.add(
       type="default"
     />
   )
+);
+
+stories.add(
+  'User (Avatar no pill)',
```

Comment:
Can we just name this to 'Avatar Without Pill' ? It can be used for anything. User is just an example usecase.

Reply:
Yup.. so how do I do that? Just make the changes and add/commit/push again?

Reply:
yup exactly

Reply:
yup exactly